src/main/java/frc/robot/Constants.java

```diff
@@ -79,14 +79,33 @@ public static class AbsoluteEncoderConstants {
   }
 
   public static class MotorConstants {
-    public static final int kIntakeDeployMotorDeviceId = 16;
-    public static final int kIntakeRollerMotorDeviceId = 9;
-    public static final int kIntakeIndexMotorDeviceId = 15; 
+    public static final int AlgaeSparkMAXDeviceId = 17;
+    public static final int AlgaeServoDeviceId = 1;
   }
 
   public static class IntakeConstants {
     public static final double kDeployPosition = 12;
     public static final double kRetractPosition = 0;
     public static final double kP = 0.04;
   }
+
+  //Algae Constants
+  public static enum IntakePosition {
```

Comment:
This is currently used to set servo angle. So, make retract 0 and deploy is 60 (in degrees)